public/js/graph/react_node.js.jsx

```diff
@@ -0,0 +1,113 @@
+function renderReactGraph() {
+    React.render(
+        <ReactSVG width={1195} height={650}/>,
+        document.getElementById('react-graph')
+    );
+}
+
+function getAttributes(node_named_map) {
+    var attrs = [];
+    //Traversing a NodeNamedMap type
+    Array.prototype.slice.call(node_named_map).forEach(function(item) {
```

Comment:
Alright, try doing 

```
for (var i = 0; i < node_named_map.length; i++) {
  var item = node_named_map[i];
  attrs[item.name] = item.value;
```

You should be able to loop through NamedNodeMaps like an array.